databases/mysql56/files/patch-innodb_engine-fcommon.diff

```diff
@@ -0,0 +1,12 @@
+Workaround for https://bugs.mysql.com/bug.php?id=100217
+--- a/plugin/innodb_memcached/innodb_memcache/CMakeLists.txt
++++ b/plugin/innodb_memcached/innodb_memcache/CMakeLists.txt
+@@ -38,7 +38,7 @@
+ STRING(REGEX REPLACE "-Wdeclaration-after-statement" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
+ ENDIF(CMAKE_C_FLAGS MATCHES "-Werror")
+ 
+-SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CMAKE_SHARED_LIBRARY_C_FLAGS} -std=gnu99")
++SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${CMAKE_SHARED_LIBRARY_C_FLAGS} -std=gnu99 -fcommon")
```

Comment:
Can this be accomplished by adding `-fcommon` to the appropriate `configure.XXXflags`, to avoid a patch? Ditto for the other one.

Reply:
My understanding is that `-fcommon` is considered a last resort workaround, so I used a patch to apply it only for the affected step in the build. It nearly sounds as if `-fno-common` behavior is so desirable that it should be put in `configure.cflags` for reproducibility with compilers lacking `-fno-common` by default.